### DISCUSSION TITLE
osu!lazer android change file location to sdcard

### DISCUSSION THREAD

**Discussion by Fxhsrx** [ppy/osu#27534] [Ideas]
this idea was already brought up btw but it's unanswered: https://github.com/ppy/osu/discussions/19883
 
i think it would be really nice to have this feature considering you can do this on other platform(s) and helps people with low internal storage space.

the idea is that the sh.ppy.osulazer on the `/storage/emulated/0/android/data/` could be moved to `/storage/(sdcard-uuid-i-think)/android/data/` and the game will access that.

internal storage:
![Screenshot_2024-03-08-19-17-24-26](https://github.com/ppy/osu/assets/116128416/d96faf1f-56b5-40e8-873d-8fbdd2fc6a3b)
to external storage:
![Screenshot_2024-03-08-19-17-46-82](https://github.com/ppy/osu/assets/116128416/22fe4b9f-802a-4224-9198-4719f0db6a65)

**Comment by BlockHead007** [comment]
I tried moving it using inbuild method of android apps settings. It reports now that all data is on External storage, but when I looked into data folder all I saw is that it is actually empty, and all the data is still on inbuild storage. I guess we need official lazer support for this.
